dev/AppNotifications/NotificationProperties.cpp

```diff
@@ -131,8 +134,11 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationProperties::get_ExpiresOnReboot(_Ou
 STDMETHODIMP_(HRESULT __stdcall) NotificationProperties::get_ToastProgressData(_Out_ ToastABI::IToastProgressData** progressData) noexcept
 {
     auto lock{ m_lock.lock_shared() };
-
-    m_toastProgressData.copy_to(progressData);
+    *progressData = nullptr;
```

Comment:
Nit: this is not needed to happen inside the lock.

Reply:
Fixing!